builder/layouts/tag.hbs

```diff
@@ -1,15 +1,14 @@
-{{#> head }}
-  <title>Tag: {{ tag }}</title>
-{{/head }}
-
-{{#> full-body }}
-
-  <div class="card">
-    <h2>Tag: {{ tag }}</h2>
-  </div>
-
-  {{#each pagination.files}}
-    {{>snippet}}
-  {{/each}}
-
-{{/full-body}}
+{{#> head}}
+    <title>✨ {{capitalize tag}} - {{site.title}} </title>
```

Comment:
we need to be consistent with `h2` below. I'm not sure the capitalization is the right choice, for tags like `Ng-content` it looks weird.

Reply:
dropped